### ISSUE TITLE
Intranet production service is not sending emails

### ISSUE THREAD

**Issue by tatyree** [ministryofjustice/cloud-platform#100]
Vinh Nguyen [11:01 AM]
Hi, I work on the Internet Team. There seems to be a issue on Live site, not sending out emails. 
We have docker setup on our local, the emails are working fine there. 
Is there a way we can check if the server is sending out the emails? If not any errors it’s bringing up?

**Comment by tatyree** [comment]
@vinhnguyen0429 - Production cluster uses sendgrid.  I'm investigating now.  You might want to check with Nathan about the status of the sendgrid account.

**Comment by tatyree** [comment]
The login details for sendgrid for intranet in rattic appear to be out-of-date.  I've slacked Nathan to see if he has more current details.

**Comment by tatyree** [comment]
According to the logs, the app is failing to connect to the smtp sendgrid server.  No further information is being recorded.  Due to the way the log groups are being stored in Cloudwatch, it is very difficult to tell when the problem started.

**Comment by tatyree** [comment]
This appears to be a sendgrid issue.  Blocked waiting on @vinhnguyen0429 to get me login details to their account (in rattic).

**Comment by tatyree** [comment]
@vinhnguyen0429 It turns out that rattic had a trailing whitespace in the password field that was being copied.  The rattic login details are good (https://rattic.service.dsd.io/cred/detail/504/), continuing to investigate.

**Comment by tatyree** [comment]
Right, and that's a dormant account that hasn't been used for over a year.  Looking for the correct account–I've asked Ollie Treend if he knows which account it might be using.

**Comment by tatyree** [comment]
Have found the correct login for the intranet TP account (https://rattic.service.dsd.io/cred/detail/1071/).  This appears to be either an outdated configuration or a billing issue with sendgrid, so it is out of scope for Platform. 

I've discussed it with @brown-a2 and handed it over to him.  Closing.
